Bottom: 5f4b653e5c2c822bc2547daef83bf9e0d463868c
Top:    b6ee76c82b801fd0006ff484bf23f355a66851d6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-14 00:18:05 +0000

Refresh of temporarily-disable-rubber

---

diff --git a/src/move.c b/src/move.c
index 6282e9e..9a4ce64 100644
--- a/src/move.c
+++ b/src/move.c
@@ -926,7 +926,8 @@ MoveObjectAndRubberband (int Type, void *Ptr1, void *Ptr2, void *Ptr3,
     if (!found_line_at_1_end)
       max_multiplier = MAX (max_multiplier, 1.0);
 
-  if (Type == LINE_TYPE)
+//  if (Type == LINE_TYPE)
+    if (0)
     {
       Coord ldx, ldy;
       ldx = (moving_line->Point2.X - moving_line->Point1.X);
